packages/safe-core-sdk/tests/createTransaction.test.ts

```diff
@@ -147,7 +147,7 @@ describe('Transactions creation', () => {
           safeAddress: safe.address,
           contractNetworks
         })
-        const safeTxGas = 111
+        const safeTxGas = 0
```

Comment:
Why this change? For this test it should be irrelevant if it's 0 or 111 ("should return a transaction with defined safeTxGas if safeVersion<1.3.0")

Reply:
I think I get it but i'd a add a separate test for handling falsy values

Reply:
Yeah, I was thinking its the edge value that caused the issue so rather have a test for it but I will add another.